resources/views/components/imgix.blade.php

```diff
@@ -0,0 +1,10 @@
+<picture>
+    <source type="image/webp" srcset="{{ $srcSet('webp') }}"/>
+    <img
+        src="{{ $src() }}"
+        srcset="{{ $srcSet() }}"
+        @if($width) width="{{ $width }}" @endif
+        @if($height) height="{{ $height }}" @endif
+        loading="lazy"
```

Comment:
The `{{ $attributes }}` are missing here - right now it's impossible to define a class for the image or something like an `alt` attribute.

Reply:
I wasn't aware that the `$attributes` variable holds the component attributes automatically - just saw this now in the [Laravel docs](https://laravel.com/docs/8.x/blade#component-attributes).

So that makes everything completely clear. 😅